docs/polymorfisme.md

```diff
@@ -0,0 +1,257 @@
+# Toepassen van polymorfisme binnen de BRP Bevraging API
+
+In de Haal Centraal BRP API worden data types gebruikt om meerdere gelijksoortige data te representeren. Voorbeelden van dit soort data types zijn de Verblijfplaats type en DatumOnvolledig type. Vanwege zijn simpele aard is het DatumOnvolledig type gebruikt om de issues en de gekozen oplossing te illustreren.
+
+Het DatumOnvolledig type wordt gebruikt om vier soorten datums te representeren:
+- 'volledige' datum. Jaar, maand en dag zijn bekend
+- 'jaar maand' datum. Jaar en maand zijn bekend, dag is onbekend
+- 'jaar' datum. Jaar is bekend, maand en dag zijn onbekend
+- 'volledig onbekende' datum. Jaar, maand en dag zijn onbekend
+
+De DatumOnvolledig type is met behulp van de OpenApi Specificatie als volgt gedefinieerd:
```

Comment:
erbij vermelden dat dit de specificatie is in versie 1.x?

Reply:
done